src/plugins/laya-la-scmc/view.vue

```diff
@@ -303,25 +305,16 @@ export default {
      *
      * Author: core
      *
-     * Last Updated: March 19, 2021
+     * Last Updated: September 28, 2022 by nv
      */
     diffSolution () {
       for (let i = 0; i < this.options.length; ++i) {
-        // is solution ?
-        if (this.solutions.includes(i)) {
-          // is also answer ?
-          if (this.answers.includes(i)) {
-            this.eval[i] = { 'far fa-check-circle text-success': true }
-          } // else {
-          // this.eval[i] = { 'far fa-times-circle text-danger': true }
-          // }
-        } else {
-          // but is answer ?
-          if (this.answers.includes(i)) {
-            this.eval[i] = { 'far fa-times-circle text-danger': true }
-          }// else {
-          // this.eval[i] = { 'far fa-check-circle text-success': true }
-          // }
+        if (this.solutions.includes(i) && this.answers.includes(i)) {
+          // is correct answer ?
+          this.eval[i] = { 'far fa-check-circle text-success': true }
+        } else if (this.answers.includes(i)) {
+          // is wrong answer ?
+          this.eval[i] = { 'far fa-times-circle text-danger': true }
```

Comment:
If no answer is is given, no item will be marked correct or wrong. In this case, I would suggest a cross mark at the correct answer.